aws_iam_policies/aws_iam_policy_role_mapping.py

```diff
@@ -16,12 +18,13 @@ def policy(resource):
         return True
 
     # Check if this policy is attached to any roles
-    if resource['Entities']['PolicyRoles'] is None:
+    if deep_get(resource, 'Entities', 'PolicyRoles') is None:
         return False
 
     # Build the list of role names this policy is actually attached to
     roles_attached = [
-        role['RoleName'] for role in resource['Entities']['PolicyRoles']
+        role['RoleName']
+        for role in deep_get(resource, 'Entities', 'PolicyRoles', default=[])
```

Comment:
This usage of deep_get will raise a `TypeError` when returning the default `default=[]` since list indices must be ints or slices, not `str`

Reply:
Just now I added a test case where `Entities` is `null`, and it seems to work fine. I believe you're confusing what the default value is being used for here. The default value `[]` is what gets iterated over to find `role`, it's not the value that `role` gets by default. All the same, good to have a test case for this so I've added one and pushed it up.